torchx/specs/named_resources_aws.py

```diff
@@ -45,161 +46,134 @@
 GiB: int = int(1024 * MEM_TAX)
 
 
+@dataclass
+class AwsResource(Resource):
```

Comment:
What you had originally was fine as long as you document that `K8S_ITYPE` is used in the aws_batch_scheduler to set the instance type for non-homogeneous CEs that have instances with resource specs that only differ for accelerators.